client/state/selectors/test/get-plugin-upload-error.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { getPluginUploadError } from 'state/selectors';
+
+const siteId = 77203074;
+const error = {
+	error: 'folder_exists',
+	message: 'folder_exists',
+};
+
+describe( 'getPluginUploadError', () => {
+	it( 'should return null by default', () => {
+		const state = deepFreeze( {
+			plugins: {
+				upload: {
+					uploadError: {}
+				}
+			}
+		} );
+		expect( getPluginUploadError( state, siteId ) ).to.be.null;
+	} );
+
+	it( 'should return current value for site', () => {
```

Comment:
Is it worth to add a test case where state has an error for another site, to verify that the selector is not incorrectly returning an error for another site?

This question applies to the rest of the tests.